test/CMakeLists.txt

```diff
@@ -7,6 +7,8 @@
 # this.
 set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
 
+# don't install googletest
+set(INSTALL_GTEST OFF)
```

Comment:
this has been bugging since ages and turns out even arch's `abseilcpp` installs gtest system-wide :roll_eyes: